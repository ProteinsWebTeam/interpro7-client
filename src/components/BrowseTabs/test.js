// @flow
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { BrowseTabsWithoutData } from '.';

const renderer = new ShallowRenderer();

describe('<BrowseTabsWithoutData />', () => {
  test('should render', () => {
    renderer.render(
      <BrowseTabsWithoutData
        data={{
          loading: false,
          payload: null,
        }}
      />,
    );
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
