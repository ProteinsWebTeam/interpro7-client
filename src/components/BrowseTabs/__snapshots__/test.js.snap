// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrowseTabsWithoutData /> should render 1`] = `
<ul
  className="tabs pp-browse-tabs"
>
  <li
    className="tabs-title"
  >
    <BrowseTabsLink
      data={
        {
          "loading": false,
          "payload": null,
        }
      }
      isFirstLevel={true}
      name="By InterPro"
      to={[Function]}
    />
  </li>
  <li
    className="tabs-title"
  >
    <BrowseTabsLink
      data={
        {
          "loading": false,
          "payload": null,
        }
      }
      isFirstLevel={true}
      name="By Member DB"
      to={[Function]}
    />
  </li>
  <li
    className="tabs-title"
  >
    <BrowseTabsLink
      data={
        {
          "loading": false,
          "payload": null,
        }
      }
      isFirstLevel={true}
      name="By Protein"
      to={[Function]}
    />
  </li>
  <li
    className="tabs-title"
  >
    <BrowseTabsLink
      data={
        {
          "loading": false,
          "payload": null,
        }
      }
      isFirstLevel={true}
      name="By Structure"
      to={[Function]}
    />
  </li>
  <li
    className="tabs-title"
  >
    <BrowseTabsLink
      data={
        {
          "loading": false,
          "payload": null,
        }
      }
      isFirstLevel={true}
      name="By Taxonomy"
      to={[Function]}
    />
  </li>
  <li
    className="tabs-title"
  >
    <BrowseTabsLink
      data={
        {
          "loading": false,
          "payload": null,
        }
      }
      isFirstLevel={true}
      name="By Proteome"
      to={[Function]}
    />
  </li>
  <li
    className="tabs-title"
  >
    <BrowseTabsLink
      data={
        {
          "loading": false,
          "payload": null,
        }
      }
      isFirstLevel={true}
      name="By Clan/Set"
      to={[Function]}
    />
  </li>
</ul>
`;
